Gruntfile.js

```diff
@@ -315,34 +257,27 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-copy');
 
   // 3rd party Grunt tasks
-  grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-release');
   grunt.loadNpmTasks('grunt-s3');
   grunt.loadNpmTasks('grunt-gitinfo');
   grunt.loadNpmTasks('grunt-sri');
 
   // Build tasks
-  grunt.registerTask('_prep', ['clean', 'gitinfo', 'version']);
-  grunt.registerTask(
-    'browserify.core',
-    ['_prep', 'browserify:core'].concat(browserifyPluginTaskNames)
-  );
-  grunt.registerTask('browserify.plugins-combined', [
+  grunt.registerTask('_prep', ['gitinfo', 'version']);
+  grunt.registerTask('build.test', ['_prep', 'rollup:core', 'rollup:test']);
+  grunt.registerTask('build.core', ['_prep', 'rollup:core', 'sri:dist']);
+  grunt.registerTask('build.plugins', [
     '_prep',
-    'browserify:plugins-combined'
-  ]);
-  grunt.registerTask('build.test', ['_prep', 'browserify.core', 'browserify:test']);
-  grunt.registerTask('build.core', ['browserify.core', 'uglify', 'sri:dist']);
-  grunt.registerTask('build.plugins-combined', [
-    'browserify.plugins-combined',
-    'uglify',
+    'rollup:plugins',
     'sri:dist',
     'sri:build'
   ]);
-  grunt.registerTask('build', ['build.plugins-combined']);
-  grunt.registerTask('dist', ['build.core', 'copy:dist']);
+  grunt.registerTask('build', ['build.core', 'build.plugins', 'uglify']);
```

Comment:
Can we utilize https://github.com/TrySound/rollup-plugin-uglify for this?

Reply:
I didn't use it because I couldn't get the plugin to output the normal **and** the uglified build.

I don't if that is possible with this plugin.